modules/server/src/main/scala/higherkindness/mu/rpc/server/metrics/MetricsServerInterceptor.scala

```diff
@@ -40,79 +39,91 @@ case class MetricsServerInterceptor[F[_]: Clock](
 
     val methodInfo: GrpcMethodInfo = GrpcMethodInfo(call.getMethodDescriptor)
 
-    E.toIO(
+    disp.unsafeRunSync {
```

Comment:
that's why we need to thread a `Dispatcher` everywhere?

Reply:
More or less, yes. Anywhere that previously used `Effect` in order to `unsafeRun*` something, will now use `Dispatcher`.

https://typelevel.org/cats-effect/docs/migration-guide#effect-concurrenteffect-synceffect